arches_her/migrations/0001_initial.py

```diff
@@ -0,0 +1,572 @@
+            helptext=None,
+            datatype="bngcentrepoint"
+        )
+
+        Widget.objects.update_or_create(
+            widgetid="31bc729d-6126-4301-8ec1-d6c4d98c68f8",
+            name="photo-widget",
+            datatype="file-list",
+            component="views/components/widgets/photo",
+            helptext=None,
+            defaultconfig={
+                "maxFilesize": "200",
+                "acceptedFiles": "",
+                "rerender": True
+            }
+        )
+    
+    def remove_widgets(apps, schema_editor):
+        Widget = apps.get_model("models", "Widget")
+
+        for widget in Widget.objects.filter(pk__in=[
+            "bcae8e90-09f7-4ae3-906b-7c7bb71a6ddf",
+            "31bc729d-6126-4301-8ec1-d6c4d98c68f8",
+        ]):
+            widget.delete()
+
+
+    def add_datatypes(apps, schema_editor):
+        Datatype = apps.get_model("models", "DDataType")
+        Widget = apps.get_model("models", "Widget")
+
+        bngpoint=Widget.objects.get(pk="bcae8e90-09f7-4ae3-906b-7c7bb71a6ddf")
+
+        Datatype.objects.update_or_create(
+            datatype="bngcentrepoint",
+            iconclass="fa fa-location-arrow",
+            modulename="bngcentrepoint.py",
+            classname="BNGCentreDataType",
+            defaultwidget=bngpoint,
+            defaultconfig=None,
+            configcomponent="views/components/datatypes/bngcentrepoint",
+            configname="bngcentrepoint-datatype-config",
+            isgeometric=False,
+            issearchable=True,
+        )
+
+    def remove_datatypes(apps, schema_editor):
+        Datatype = apps.get_model("models", "DDataType")
+
+        for datatype in Datatype.objects.filter(datatype__in=[
+            "bngcentrepoint",
+        ]):
+            datatype.delete()
+
+
+    def add_search_components(apps, schema_editor):
+        SearchComponent = apps.get_model("models", "SearchComponent")
+
+        SearchComponent.objects.update_or_create(
+            searchcomponentid="25ca3536-9eb4-4fd5-b2a5-badfd9a266de",
+            name="BNG Filter",
+            icon="fa fa-compass",
+            modulename="bng-filter.py",
+            classname="BngFilter",
+            type="popup",
+            componentpath="views/components/search/bng-filter",
+            componentname="bng-filter",
+            sortorder="0",
+            enabled=True,
+        )
+    
+    def remove_search_components(apps, schema_editor):
+        SearchComponent = apps.get_model("models", "SearchComponent")
+
+        for search_component in SearchComponent.objects.filter(pk__in=[
+            "25ca3536-9eb4-4fd5-b2a5-badfd9a266de",
+        ]):
+            search_component.delete()
+
+
+    add_map_source = """
+        INSERT INTO map_sources(name, source)
+            VALUES ('select-application-area', '{
+                "data": "/geojson?nodeid=1909956f-3a3b-11eb-ae99-f875a44e0e11&include_geojson_link=true",
+                "type": "geojson"
+            }');
```

Comment:
This was recently changed for the new application area MVT endpoint

```suggestion
INSERT INTO map_sources(name, source)
    VALUES ('select-application-area', '{
        "data": "/geojson?nodeid=1909956f-3a3b-11eb-ae99-f875a44e0e11", 
        "type": "vector", 
        "tiles": ["/application-areas/{z}/{x}/{y}.pbf"], 
        "minzoom": 6
    }');
```